src/node/progress_tracker.h

```diff
@@ -505,9 +505,32 @@ namespace ccf
       return hash;
     }
 
+    kv::Consensus::SeqNo get_highest_commit_level()
```

Comment:
I think a name like `committed_nonce_watermark` or `highest_committed_nonce` would be more descriptive.